src/lints/trait_removed_supertrait.ron

```diff
@@ -0,0 +1,60 @@
+SemverQuery(
+    id: "trait_removed_supertrait",
+    human_readable_name: "supertrait removed or renamed",
+    description: "It is a breaking change to loosen generic bounds on a type since this can break users expecting the tightened bounds.",
+    required_update: Major,
+    reference_link: None,
```

Comment:
Any chance we can find something to cite here?

Reply:
I should have asked about this earlier, on the `semver` reference does not seem to be an entry for this lint.

There is [this](https://doc.rust-lang.org/cargo/reference/semver.html#minor-loosening-generic-bounds) that is somewhat similar but it's definitely not the exact case nor does the explanation help (IMHO).

The [documentation](https://doc.rust-lang.org/reference/items/traits.html#supertraits) about supertraits helps to understand what is breaking indirectly.

Is it worth adding any of those links?

Reply:
Yes, the docs link for supertraits is the best option. It's not perfect, but it's certainly better than nothing.

Reply:
Thank you, I will add it :)